skonfig/__init__.py

```diff
@@ -104,8 +107,12 @@ def _set_conf_dirs(configuration):
     else:
         configuration["conf_dir"] = []
     configuration["conf_dir"].insert(0, SKONFIG_CONFIGURATION_DIRECTORY)
+
+    # find sets and insert them into PATH
     sets_dir = os.path.join(SKONFIG_CONFIGURATION_DIRECTORY, "set")
     if os.path.isdir(sets_dir):
-        for set_dirname in os.listdir(sets_dir):
-            configuration["conf_dir"].append(os.path.join(sets_dir, set_dirname))
+        configuration["conf_dir"] += [
+            os.path.join(sets_dir, s) for s in os.listdir(sets_dir)
+        ]
+
```

Comment:
Matter of taste, but to keep same style, let's drop empty lines in functions? Here and on line 110, before comment.